src/scripts/config/main.ts

```diff
@@ -89,5 +89,104 @@ export default {
   legendImage: `${baseUrlTiles}/{id}/legend.png`,
   downloadUrls,
   localStorageLanguageKey: 'language',
-  delay: 5000
+  delay: 5000,
+  landCoverLegendValues: [
```

Comment:
The main config file is not the right place for the land covers legend data. It should be provided by the layers metadata which is loaded from cloud storage